portfolio/src/main/webapp/script.js

```diff
@@ -34,3 +34,16 @@ function gallery(){
 
     galleryContainer.style.display = "block";
 }
+
+function createMap() {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      //{center: {lat: 37.422, lng: -122.084}, zoom: 16});
+      {center: {lat: 18.1096, lng: -77.2975}, zoom: 10});
+}
+
+const trexMarker = new google.maps.Marker({
+    position: {lat: 18.1096, lng: 77.2975},
+    map: map,
```

Comment:
Is this code working? Since it's just a top-level const in this javascript file, I'd expect it to run when script.js is loaded, which should be before the createMap() function actually gets called and creates the map const that this code refers to.

If it's already working, great. Otherwise, I'd try putting it at the end of the createMap function.